platform-ui-skin/src/main/webapp/skin/less/core/helpers.less

```diff
@@ -97,6 +97,9 @@
 .text-font-size {
   font-size: @fontSizeLarge !important;
 }
+.text-font-size-large {
+  font-size: @fontSizeXlarge !important;
+}
```

Comment:
Use CSS Helpers of Vuetify : `display-1`, `display-2` ...

Reply:
done